src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -394,7 +394,7 @@ void BulletRigidObject::constructAndAddRigidBody(MotionType mt) {
         bObjectRigidBody_->getCollisionFlags() |
         btCollisionObject::CF_KINEMATIC_OBJECT);
     CORRADE_INTERNAL_ASSERT(bObjectRigidBody_->isKinematicObject());
-    CORRADE_INTERNAL_ASSERT(!bObjectRigidBody_->isStaticObject());
+    CORRADE_INTERNAL_ASSERT(bObjectRigidBody_->isStaticObject());
```

Comment:
Can the rigid body be both kinematic and static?

Reply:
The Bullet static flag is set when you provide 0 mass, so yes it can be both.

Reply:
The won't change the Habitat MotionType of the object or what we allow it to do.